.travis.yml

```diff
@@ -3,21 +3,26 @@ sudo: false
 env:
   global:
     - RUST_BACKTRACE=1
-    - CLIPPY_VERSION=0.0.179
+    - CLIPPY_VERSION=0.0.192
     - RUSTFLAGS="-D warnings"
 cache: cargo
 
-# `allow_failures` requires this, but we set it below.
 rust:
 
 matrix:
   include:
+  # This build uses stable and checks rustfmt (clippy is not on stable).
   - rust: stable
-  - rust: nightly-2018-01-12
+    install:
+      - rustup component add rustfmt-preview
+  # This build uses the nightly used by TiKV and checks both rustfmt & clippy.
+  - rust: nightly-2018-04-06
     install:
       - export PATH="$PATH:$HOME/.cargo/bin"
+      - rustup component add rustfmt-preview
```

Comment:
What if the format is different from stable channel?

Reply:
:+1: Great catch, you're totally right. We should just run `rustfmt` on `stable`.